packages/flame_test/pubspec.yaml

```diff
@@ -2,19 +2,21 @@ name: flame_test
 description: A package with classes to help testing applications using Flame
 version: 1.1.0
 homepage: https://github.com/flame-engine/flame/tree/main/packages/flame_test
+publish_to: none
```

Comment:
Here as well, I don't think these changes are correct (except the test package bump)